packages/components/src/mobile/bottom-sheet/range-cell.native.js

```diff
@@ -130,49 +134,60 @@ class BottomSheetRangeCell extends Component {
 			cellProps.label, value
 		);
 
+		const defaultSliderStyle = getStylesFromColorScheme( styles.sliderTextInput, styles.sliderDarkTextInput );
+
 		return (
 			<Cell
 				{ ...cellProps }
+				cellContainerStyle={ styles.cellContainerStyles }
+				cellRowContainerStyle={ styles.cellRowStyles }
 				accessibilityRole={ 'none' }
 				editable={ false }
 				accessible={ accessible }
 				onPress={ this.onCellPress }
 				accessibilityLabel={ accessibilityLabel }
+				allowReset={ this.handleReset }
```

Comment:
It's interesting that the root `Cell` component provides `Reset` option, but the reset functionality has to be handled by the parent component. It feels to me more like a generic button that could be anything. Maybe we can add it as an accessory view on this component, similar how we added the Switch?

Just by curiosity, I added `allowReset` to other controls, it shows the Reset button but it looks a bit off (apart than it does nothing). It doesn't feel to me that it deserves to be on the root Cell component, unless we plan to add reset to more controls? (In which case this might be a good-enough first step). cc @iamthomasbishop 

![IMG_3459](https://user-images.githubusercontent.com/9772967/68311193-7fb4fb00-00b1-11ea-9751-bb591265d1c8.png)

Reply:
> It doesn't feel to me that it deserves to be on the root Cell component, unless we plan to add reset to more controls? (In which case this might be a good-enough first step)

That's correct, I don't think there are any other planned components at the moment that will need an explicit `reset` button like this.

Reply:
One more note: if we do end up adding `reset` to other cells/components, we'll want to display them in the same two-row way which provides more space for the `reset` button.

Reply:
Currently we don't have many `settings` in block but maybe it's not bad idea to allow reset single field. Correct me if I'm wrong but it seems now it's possible to manually delete/change the value or clear all settings.

Reply:
* Reset button is placed within a `Cell` component, because we want to locate it next to the label (_Height in pixels_) which is defined there actually, otherwise it has to be positioned absolutely.

* Also we can block `allowReset` button option for other controls to avoid the case